CMakeLists.txt

```diff
@@ -14,6 +14,10 @@
 
 cmake_minimum_required(VERSION 3.23.1 FATAL_ERROR)
 
+# TODO(keith): REMOVE BEFORE MERGING
+set(rapids-cmake-repo "kkraus14/rapids-cmake")
+set(rapids-cmake-branch "spdlog_1.11_fmt")
```

Comment:
Just pointing this out: temporarily pointed it to the branch behind https://github.com/rapidsai/rapids-cmake/pull/368 so we can test more easily before merging both.

Reply:
Reverted in #1209.